.github/actions/cdn_assets/action.yml

```diff
@@ -21,15 +21,15 @@ runs:
       working-directory: ${{ inputs.working-directory }}
       run: |
         curl https://${{ inputs.cdn }}/cli/${{ inputs.assets_version }}.tar.gz --output assets.tar.gz --fail
-        curl https://${{ inputs.cdn }}/cli/pathfinder/static.tgz --output static.tgz --fail
-        tar -zxvf static.tgz
-        rm static.tgz
-        gunzip assets.tar.gz
-        tar -rf assets.tar static
-        rm -r static
-        gzip assets.tar
         cp assets.tar.gz crates/server/assets
         mkdir ~/.grafbase
         cp assets.tar.gz ~/.grafbase
         (cd ~/.grafbase && tar -xf assets.tar.gz)
         touch ~/.grafbase/version.txt
+
+    - name: Build cli-app
+      shell: bash
+      working-directory: './packages/cli-app'
+      run: |
+        npx pnpm i
+        npx pnpm run cli-app:build
```

Comment:
Should we rename the action?

Reply:
Yes (or do the CLI build outside of this)

Reply:
done in [3048de1](https://github.com/grafbase/grafbase/pull/819/commits/3048de138b1dba9e1ed9d8c40c82e7a1098afe78)